<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MusicManagementClass</class>
 <widget class="QMainWindow" name="MusicManagementClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>990</width>
    <height>526</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MusicManagement</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTableWidget" name="MasterList">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>601</width>
      <height>351</height>
     </rect>
    </property>
    <property name="gridStyle">
     <enum>Qt::NoPen</enum>
    </property>
    <property name="columnCount">
     <number>6</number>
    </property>
    <column>
     <property name="text">
      <string>I D</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Singer</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Genre</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Add</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Info</string>
     </property>
    </column>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>430</y>
      <width>195</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="push_add">
       <property name="text">
        <string>Add Song</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="push_make_empty">
       <property name="text">
        <string>Make Empty</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTableWidget" name="ResultList">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>70</y>
      <width>351</width>
      <height>351</height>
     </rect>
    </property>
    <property name="columnCount">
     <number>4</number>
    </property>
    <column>
     <property name="text">
      <string>I D</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Singer</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Genre</string>
     </property>
    </column>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>430</y>
      <width>341</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>   I D</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="edit_delete"/>
     </item>
     <item>
      <widget class="QPushButton" name="push_delete">
       <property name="text">
        <string>Delete Song</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="push_modify">
       <property name="text">
        <string>Modify Song</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>205</width>
      <height>63</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLineEdit" name="edit_fstream"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="push_open">
         <property name="text">
          <string>Open</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="push_save">
         <property name="text">
          <string>Save</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>0</y>
      <width>351</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QComboBox" name="search_type">
         <item>
          <property name="text">
           <string>Search</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>ID</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Name</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Singer</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Genre</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Lyrics</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Tag</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="edit_search"/>
       </item>
       <item>
        <widget class="QPushButton" name="push_search">
         <property name="text">
          <string>Search</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="push_recent">
         <property name="text">
          <string>Recent Song</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="push_priority">
         <property name="text">
          <string>Priority Song</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="push_recommend">
         <property name="text">
          <string>Recommend</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_5">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>420</y>
      <width>195</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QPushButton" name="push_playlist">
       <property name="text">
        <string>Play List</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="push_exit">
       <property name="text">
        <string>Exit</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>20</y>
      <width>1001</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>My Contents Management</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>990</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="MusicManagement.qrc"/>
 </resources>
 <connections/>
</ui>
